tasks/parse-pdr/schemas/config.json

```diff
@@ -8,13 +8,26 @@
       "type": "string"
     },
     "stack": {
-      "description": "The name of the Task's CloudFormation Task, useful as a prefix",
+      "description": "The cumulus stack prefix, pulled from $.meta.stack",
       "type": "string"
     },
     "granuleIdFilter": {
       "description": "A regular expression that is applied against the filename of the first file of each granule contained in the PDR.  If the regular expression matches, then the granule is included in the output.  Defaults to '.', which will match all granules in the PDR.",
       "type": "string"
     },
+    "uniquifyGranuleId": {
+      "description": "A flag (boolean or string) that indicates if the lambda should uniquify the granule IDs found in the PDR. Accepts true/false or 'true'/'false'. Defaults to false.  Positive string value is 'true'",
+      "anyOf": [
+        { "type": "boolean" },
+        { "type": "string", "enum": ["true", "false"] }
+      ],
+      "default": false
+    },
+    "hashLength": {
```

Comment:
Can the name be more specific, like uniquifyHashLength?
Also add the new parameter to README.md

Reply:
We can, however I was attempting to keep this consistent between the top level tasks and we have it set in add-unique-granuleId.         Thoughts?     We should probably low-key get weigh in from @npauzenga  if we're going to change that decision.

Reply:
Yeah, I could sorta go either way. I'm leaning towards `hashLength` because we have the description.